scripting/get5/mapveto.sp

```diff
@@ -14,6 +14,15 @@ public void CreateVeto() {
   g_VetoCaptains[MatchTeam_Team1] = GetTeamCaptain(MatchTeam_Team1);
   g_VetoCaptains[MatchTeam_Team2] = GetTeamCaptain(MatchTeam_Team2);
   ResetReadyStatus();
+  if (g_FreezePlayersVetoCvar.BoolValue)
+  {
+    Pause();
+    for (int i = 1; i <= MaxClients; i++) {
```

Comment:
For the sake of reusing what's existing, you could use `LOOP_CLIENTS(i) {` instead of `for (int i = 1; i <= MaxClients; i++) {`.